### DISCUSSION TITLE
Cannot run start --data / cannot find pm2/lib/ProcessContainerFork.js

### DISCUSSION THREAD

**Discussion by borecz** [mockoon/mockoon#1023] [Help]
**Describe the bug**
`npx mockoon-cli start --data open-api-specs/foodkoala.json`

`Error: Cannot start mockoon-foodkoala-api due to errors`

```
Node.js v18.15.0
node:internal/modules/cjs/loader:1078
  throw err;
  ^

Error: Cannot find module '/Users/WORK/S1/prism-react-swagger-poc/node_modules/pm2/lib/ProcessContainerFork.js'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1075:15)
    at Module._load (node:internal/modules/cjs/loader:920:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
    at node:internal/main/run_main_module:23:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
```

package.json

```
  "devDependencies": {
    "@mockoon/cli": "2.4.0"
  },
```

<img width="214" alt="image" src="https://user-images.githubusercontent.com/57007154/234062101-8beb771f-3b5b-40a9-a462-58efcaf8b05d.png">

**Mockoon version:**

2.4.0

**OS / OS version:**

MacOS

**Comment by 255kb** [comment]
This is not directly link to Mockoon CLI, but rather to the PM2 installation (PM2 is used to manage the CLI processes).
Could you try to reinstall the CLI by doing `npm r -g @mockoon/cli` then `npm i -g @mockoon/cli`. If this doesn't solve the issue there are a bunch of things to try [there](https://github.com/Unitech/pm2/issues/1943) (removing the `~/.pm2` folder, etc.). 
Let me know if it is working.
